roles/ceph-mds/tasks/docker/pre_requisite.yml

```diff
@@ -19,6 +19,11 @@
     state: present
   when: ansible_os_family == 'RedHat'
 
+- name: install six
```

Comment:
can you use a `with_items` loop to install pip's packages? so you can incorporate that with the docker-py installation